core/src/main/java/com/linecorp/armeria/internal/common/Http1ObjectEncoder.java

```diff
@@ -92,11 +90,6 @@ public abstract class Http1ObjectEncoder implements HttpObjectEncoder {
      */
     private int maxIdWithPendingWrites = Integer.MIN_VALUE;
 
-    /**
-     * The ID of the last request whose response headers is written.
-     */
-    private int lastResponseHeadersId;
```

Comment:
We don't have to have this information because it's only used on the server-side.
So I moved this to the server encoder.